cmd/otelcol/Dockerfile.windows

```diff
@@ -31,7 +31,7 @@ RUN Remove-Item "SignalFxAgent-win64.zip" -force
 RUN Remove-Item "agent-bundle\bin" -force -Recurse
 RUN Remove-Item "agent-bundle\etc" -force -Recurse
 RUN Get-ChildItem -include __pycache__ -recurse -force | Remove-Item -force -Recurse
-RUN Get-ChildItem -recurse -path agent-bundle\* -include *.key,*.pem | Remove-Item -force
+RUN Get-ChildItem -recurse -path agent-bundle\*test* -include *.key,*.pem | Remove-Item -force
```

Comment:
@jcheng-splunk Do you happen to know the correct way to do this?

Reply:
Maybe try something like
```
Get-ChildItem -recurse -path agent-bundle\* -include *.key,*.pem | Where-Object { $_.Directory -match 'tests' } | Remove-Item -force
```